src/main/java/com/google/sps/data/Marker.java

```diff
@@ -65,6 +71,10 @@ public String getMarkerName() {
     return markerName;
   }
 
+  public void setId() {
+    this.id = id;
```

Comment:
id should probably be passed as a parameter.

Reply:
Done, sorry